app/controllers/admin/userManagement.py

```diff
@@ -0,0 +1,34 @@
+from flask import Flask, render_template,request, flash, g, abort
+from app.models.user import User
+from app.controllers.admin import admin_bp
+from app.logic.userManagement import addCeltsAdmin,addCeltsStudentStaff,removeCeltsAdmin,removeCeltsStudentStaff
+import re
```

Comment:
I like to organize the import statements. python standard library imports should go at the top